docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -1539,6 +1539,12 @@ mimir_query_engine:
   # applies if the Mimir query engine is in use.
   # CLI flag: -querier.mimir-query-engine.enable-subqueries
   [enable_subqueries: <boolean> | default = true]
+
+  # (experimental) Enable support for classic histogram functions in Mimir's
```

Comment:
```suggestion
  # (experimental) Enable support for classic histogram functions in the Mimir
```

Reply:
We try to avoid using product names as possessives. We can say `the Mimir query engine` instead, like you already have below 😄